samples/bluetooth/peripheral_lbs/README.rst

```diff
@@ -3,22 +3,88 @@
 Bluetooth: Peripheral LBS
 #########################
 
+This sample demonstrates how to use the :ref:`lbs_readme`.
+
 Overview
 ********
 
-This sample demonstrates how to use the :ref:`lbs_readme`. When connected
-the example will send the state of button 1 on the development board
-to the mobile application. While the mobile application can control the
-state of the LED 3 on development board.
+The peripheral LBS sample demonstrates how to use the :ref:`lbs_readme`. When connected, the example sends the state of button 1 on the development board to the connected device (a phone or tablet). The mobile application on the device can display the received button state and can control the state of the LED 3 on the development board.
 
 Requirements
 ************
 
-* Any Nordic development board
-* A mobile running a compatible application (nRF Connect, nRF Blinky, nRF Toolbox)
+* One of the Nordic development boards:
+
+  * nRF52840 Development Kit board (PCA10056)
+  * nRF52 Development Kit board (PCA10040)
+  * nRF51 Development Kit board (PCA10028) - limited support; some features, like an over-the-air data rate of 2 Ms/s, are not available on this board
```

Comment:
This is  general limitation of the nrf51, i suggest to remove this.
Data rate should not affect this sample in any way, right?

Reply:
well, right - there is just exchanging of 1 byte, so the data rate limitation will not affect the behavior, I'll fix it.